generator/ServiceClientGeneratorLib/GeneratorDriver.cs

```diff
@@ -1497,55 +1497,6 @@ private static void RemoveOrphanedServices(string path, IEnumerable<string> code
             }
         }
 
-        /// <summary>
```

Comment:
Don't we need to keep this here as long as we're doing syncs from V3? Any new service will have the `_bcl35` folder for example.